project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -273,6 +273,11 @@ function c4m_user_initials($account) {
 function c4m_user_name($user) {
   $wrapper = entity_metadata_wrapper('user', $user);
 
+  // As there is no name at this stage.
+  if (isset($user->is_new)) {
```

Comment:
When creating a new user programmatically, this function runs with `$user` empty except `$user->is_new = TRUE;`